project/AppDependencies.scala

```diff
@@ -23,20 +23,19 @@ object AppDependencies {
   val test: Seq[ModuleID] = Seq(
     "com.github.tomakehurst" % "wiremock"                  % "2.27.2",
     "com.typesafe.play"      %% "play-test"                % current ,
-    "org.mockito"            % "mockito-core"              % "3.11.2",
-    "org.jsoup"              % "jsoup"                     % "1.14.2",
+    "org.mockito"            % "mockito-core"              % "4.6.1",
+    "org.jsoup"              % "jsoup"                     % "1.15.2",
     "org.pegdown"            % "pegdown"                   % "1.6.0" ,
     "org.scalatest"          %% "scalatest"                % "3.2.9" ,
     "org.scalatestplus.play" %% "scalatestplus-play"       % "5.1.0" ,
     "org.scalatestplus"      %% "mockito-3-4"              % "3.2.9.0",
     "com.vladsch.flexmark"   % "flexmark-all"              % "0.35.10",
-    "org.scalacheck"         %% "scalacheck"               % "1.15.4",
-//    "uk.gov.hmrc"            %% "http-verbs-test"          % "1.8.0-play-27",
```

Comment:
remove comment?

Reply:
Oh I see, you've highlighted my deletion of it.